pom.xml

```diff
@@ -515,7 +515,13 @@
       <dependency>
         <groupId>org.apache.parquet</groupId>
         <artifactId>parquet-avro</artifactId>
-        <version>${parquet.version}</version>
+        <version>1.8.1</version>
```

Comment:
Could you add a comment here about intentional pinning to version 1.8.1?

Reply:
+1